bot/exts/moderation/watchchannels/talentpool.py

```diff
@@ -237,13 +264,18 @@ async def unwatch(self, user_id: int, reason: str) -> bool:
     def _nomination_to_string(self, nomination_object: dict) -> str:
         """Creates a string representation of a nomination."""
         guild = self.bot.get_guild(Guild.id)
+        entries = []
+        for site_entry in nomination_object["entries"]:
+            actor_id = site_entry["actor"]
+            actor = guild.get_member(actor_id)
 
-        actor_id = nomination_object["actor"]
-        actor = guild.get_member(actor_id)
+            reason = site_entry["reason"] or "*None*"
+            created = time.format_infraction(site_entry["inserted_at"])
```

Comment:
This isn't really an infraction so maybe the name of the function should change, but this is out of scope for this PR

Reply:
This would need to replace it everywhere, so yeah, let's not do this here.